site2/website/release-notes.md

```diff
@@ -797,6 +797,116 @@ For more context about this issue, see [[Pulsar Functions and IO] Cannot upgrade
 - [C++] Remove namespace check for MultiTopicsConsumerImpl [#9520](https://github.com/apache/pulsar/pull/9520)
 
 
+### 2.7.4
+
+#### 2021-12-14
+
+#### Broker
+- Fix DispatchRateLimiter does not take effect. [#8611](https://github.com/apache/pulsar/pull/8611)
+- Fix set-publish-rate when using preciseTopicPublishRateLimiterEnable=true. [#10384](https://github.com/apache/pulsar/pull/10384)
+- fix retention policy in topic policy not work. [#11021](https://github.com/apache/pulsar/pull/11021)
```

Comment:
Please keep consistent for the following aspects: 
- verb tense (use **present tense**). reason: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp
- capitalization 
    - use sentence-style capitalization. details: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.s1g5tosy5w3i
    - capitalize proper nouns correctly. details: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.je5thtkj42cb

Besides, please pay attention to the following aspects：
- do not use contractions in technical information. reason: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.qa5rwiehb245


Can you check all occurrences overall and update them? Thanks

Reply:
All right.